### PR TITLE
fix view and width of form row plugin

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-327
- Please ignore compiled css file

Before:

screen shot 2015-08-31 at 17 39 41

After: 

screen shot 2015-09-01 at 09 24 03

### CODE REVIEW

**Comment by Brankub** [comment]
:+1:

**Comment by vxsx** [comment]
screen shot 2015-09-01 at 09 31 03

**Comment by lory87** [comment]
screen shot 2015-09-01 at 09 47 50

**Comment by vxsx** [comment]
perfect :+1:  thanks!

File: djangocms_admin_style/sass/partials/_forms.scss [hunk 1]
```diff
@@ -1,35 +1,72 @@
...
     }
     #content {
         margin-top: 20px !important;
     }
+    .bootstrap3-column {
+        .form-row {
+            overflow: hidden;
+            padding-bottom: 10px;
+            border-bottom: solid 1px $line-color-ultralight;
+             &.field-create {
```
Comment (vxsx) [inline djangocms_admin_style/sass/partials/_forms.scss]: intentation is weird here though

**Pull request merged**